src/storage-controller/src/lib.rs

```diff
@@ -1782,6 +1846,21 @@ where
             }
         }
 
+        if let Ok((id, dropped_read_hold)) = self.read_holds_rx.try_recv() {
+            let mut dropped_read_holds = BTreeMap::new();
+            dropped_read_holds.insert(id, dropped_read_hold);
```

Comment:
I was confused by this variable name because I thought the channel would only contain `ReadHold::drop` events. But it also contains downgrades and clone events, so how about calling it `changes` instead?